news/2025-02-13-conda-forge-v1-recipe-support.md

```diff
@@ -0,0 +1,84 @@
+# Announcing the new recipe format on conda-forge
+
+The conda-forge team is excited to announce that the v1 recipe format is available on conda-forge. The v1 recipe format is a community initiative dating back over 3 years to improve the recipe format for conda packages. If you are a maintainer of a feedstock on conda-forge, you have probably dealt with `meta.yaml` files that conda-build utilizes. The file format has some limitations which is why the community has come together to come up with an improved version of the format: the v1 format.
+
+## rattler-build and the v1 spec
+
+The v1 recipe format has a number of benefits:
+
+- It always parses as valid YAML which makes it much easier to modify it with the bot infrastructure of conda-forge. While meta.yaml looks like YAML, it can contain Jinja logic incompatible with the YAML specification, which make significantly complicates parsing and automated manipulation. Additionally, the recipe has a published JSON schema which means that the editing experience in VSCode is greatly improved with contextual help.
```

Comment:
```suggestion
- It always parses as valid YAML which makes it much easier to modify it with the bot infrastructure of conda-forge. While meta.yaml looks like YAML, it can contain Jinja logic incompatible with the YAML specification, which make significantly complicates parsing and automated manipulation. Additionally, the new recipe format has [a published JSON schema](https://github.com/prefix-dev/recipe-format/blob/main/schema.json) which means that the editing experience in VS Code is greatly improved with contextual help.
```

Reply:
"which make significantly" -> "which sifnificantly"